src/components/ActivitiesListItem/ActivitiesListItem.js

```diff
@@ -27,29 +30,53 @@ const ActivityDesc = styled.p`
   word-break: break-word;
 `;
 
-const ActivitiesListItem = ({ activity: { name, activityOrdinal, teacher, id, room } }) => (
-  <>
-    <ActivityOrdinalWrapper>{`${activityOrdinal}.`}</ActivityOrdinalWrapper>
-    <EditActivityLink data-test="react-link" to={EDIT_ACTIVITY_ROUTE(id)}>
-      {name}
-    </EditActivityLink>
-    <ActivityDesc>
-      <FormattedMessage
-        id="planView.activityDetails"
-        defaultMessage="Teacher: {teacher}, in room: {room}"
-        values={{ teacher, room }}
-        data-test="activity-details"
-      />
-    </ActivityDesc>
-  </>
-);
+const HighlightedSpan = styled.span`
+  color: ${props => props.isHighlighted && `${colors.pink}`};
```

Comment:
I am not the fan of naming colors by they actual color. If you will change pink to green... You will have to change many places.